components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -377,4 +380,41 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             .unwrap();
         ch.set_result(meta);
     }
+
+    // the v1's post_apply
+    // As the logic is mostly for read, rename it to handle_read_after_apply
+    pub fn handle_read_on_apply<T>(
+        &mut self,
+        ctx: &mut StoreContext<EK, ER, T>,
+        apply_res: ApplyRes,
+        progress_to_be_updated: bool,
+    ) -> bool {
+        let mut has_ready = false;
```

Comment:
We don't need that. If it needs to be marked has ready, just call `set_has_ready` is OK. It's needed in the past because `has_ready` is part of `PeerFsm`.